pages.ko/common/curl.md

```diff
@@ -0,0 +1,36 @@
+# curl
+
+> 데이터를 서버에서 혹은 서버로 전송. HTTP,FTP 및 POP3를 포함한 대부분의 프로토콜 지원. 
+> 더 많은 정보: [https://curl.haxx.se](<https://curl.haxx.se/>).
+
+- URL의 내용을 파일로 다운로드:
+
+`curl {{URL주소}} -o {{파일명}}`
+
+- URL에 표시된 파일 명으로 출력을 저장하고 파일을 다운로드:
+
+`curl -O {{URL주소/파일명}}`
+
+- [L]위치 리다이렉션 후 파일을 다운로드 하고, 자동으로 이전 파일 [C]전송(재시작):
+
+`curl -O -L -C - {{URL주소/파일명}}`
+
+- 양식 인코딩 데이터 전송(`application/x-www-form-urlencoded`유형의 POST 요청):
+
+`curl -d {{'name=bob'}} {{URL주소/양식}}`
+
+- 사용자 지정 HTTP 메서드를 사용하여 추가 헤더로 요청 전송:
+
+`curl -H {{'X-My-Header: 123'}} -X {{PUT}} {{URL주소}}`
+
+- 적절한 컨텐츠 유형 헤더를 지정하여 JSON 포멧으로 데이터 전송:
+
+`curl -d {{'{"name":"bob"}'}} -H {{'Content-Type: application/json'}} {{URL주소}}`
+
+- 서버 인증을 위한 사용자 이름 및 비밀번호 전달:
+
+`curl -u myusername:mypassword {{URL주소}}`
+
+- 리소스에 대한 클라이언트 인증서 및 키 전달, 인증서 유효성 검사 스킵:
+
+`curl --cert {{클라이언트.pem}} --key {{키.pem}} --insecure {{URL주소}}`
```

Comment:
```suggestion
`curl --cert {{클라이언트.pem}} --key {{키.pem}} --insecure {{https://example.com}}`
```
In this case it's particularly important since the URL protocol needs to be `https`.